src/boot/bless-boot.c

```diff
@@ -370,22 +363,22 @@ static int verb_status(int argc, char *argv[], void *userdata) {
                         return log_error_errno(errno, "Failed to open $BOOT partition '%s': %m", *p);
                 }
 
-                if (faccessat(fd, skip_slash(path), F_OK, 0) >= 0) {
+                if (faccessat(fd, skip_slash_or_dot(path), F_OK, 0) >= 0) {
```

Comment:
Hmm, I do not think it is necessary to drop `./` in this case and below. Maybe, `skip_leading_chars(path, "/")` is enough?

Reply:
Hmm, we pass these to `_at` functions, and `./` has no effect either?

Reply:
Right. I do not say using skip_slash_or_dot() is wrong, but that is not necessary.